ios/PurchasesHybridCommon/PurchasesHybridCommon/RCCommonFunctionality.m

```diff
@@ -313,6 +318,10 @@ + (void)invalidatePurchaserInfoCache {
     [RCPurchases.sharedPurchases invalidatePurchaserInfoCache];
 }
 
++ (void)presentCodeRedemptionSheet API_AVAILABLE(ios(14.0)) API_UNAVAILABLE(tvos, macos, watchos) {
+    [RCPurchases.sharedPurchases presentCodeRedemptionSheet];
```

Comment:
We should add the `NSAssert(RCPurchases.sharedPurchases, @"You must call setup first.");` like in the other functions

Reply:
good catch

Reply:
updated